### PR TITLE
add example for checking last permissions octet

### PR DESCRIPTION
**Pull request by thomascate**
Recently had trouble figuring out how to check the last octet permissions aka everyone/everybody/all users perms. Added example to docs.

### CODE REVIEW

File: docs/resources/file.md.erb [hunk 1]
```diff
@@ -61,6 +61,10 @@ an owner:
 
     it { should be_executable.by('owner') }
 
+any user:
```
Comment (adamleff) [inline docs/resources/file.md.erb]: Can we change this (and the others) to "any user other than the owner or members of the file's group"? "any user" is actually incorrect. If the file is mode `004`, the owner and members of the group still won't be able to read it even though the "others" have read bit enabled.

Suggestions for a shorter string than what I provided are also welcome. :)

**Review by adamleff — CHANGES_REQUESTED** [review]

**Comment by adamleff** [comment]
Also @thomascate can you please sign-off your commit to satisfy the requirements of the [DCO](https://github.com/chef/inspec/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco)? Lemme know if you need help with that.

File: docs/resources/file.md.erb (hunk 1 above)
Comment (thomascate) [reply docs/resources/file.md.erb]: Good catch, couldn't come up with a shorter string so I used yours. :)

**Review by adamleff — APPROVED** [review]
Thanks @thomascate!

**Pull request merged**